nautobot/docs/administration/nautobot-server.md

```diff
@@ -383,3 +383,13 @@ Listening on port http://localhost:9000. Stop with CONTROL-C.
 ```
 
 Please see the guide on [Troubleshooting Webhooks](../models/extras/webhook.md#troubleshooting) for more information.
+
+### `startplugin`
+
+`nautobot-server startplugin [app_name]`
+
+Create a new plugin with `app_name`.
+
+This command is just like the django-admin [startapp](https://docs.djangoproject.com/en/dev/ref/django-admin/#startapp). But has the template directory set to `nautobot/core/template/plugin_template`. This command assists with creating the Nautobot plugin folder structure.
+
```

Comment:
It'd be nice to provide an example either here or in the plugin dev docs, as it's not immediately obvious to me where the created plugin would be by default if not specifying the `directory` parameter, nor whether I should be specifying the project root directory (i.e. `~/my-plugin/` or the Python package root directory (`~/my-plugin/my_plugin/`) as a parameter.

Reply:
Sure, it defaults to whatever directory you issue the command from (which could be anywhere) and by default creates a directory with whatever the `name` input was.  So, in your example `<cwd>/my-plugin/`.